frontend/packages/ceph-storage-plugin/src/components/create-storage-system/payloads.ts

```diff
@@ -139,3 +141,29 @@ export const createExternalSubSystem = async (subSystemPayloads: Payload[]) => {
     throw err;
   }
 };
+
+const validateCRDReadiness = (crd: K8sResourceKind, plural: string) =>
+  crd?.status?.acceptedNames?.plural === plural;
+
+export const waitforCRD = async (name: string, plural: string, maxAttempts = 15) => {
```

Comment:
```suggestion
export const waitforCRD = async (model, maxAttempts = 15) => {
```
Based on my comments for https://github.com/openshift/console/pull/10375/files#diff-0b182f59d1f8f9cb38d1b95d6f766904dac5c05a3077e8e29e46f964d05c613fR177 this function signature could be simplified to the suggestion.